shuttle/templates/request_shuttle.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'shuttle_base.html' %}
+<!--TODO: Restrict roles to Bethel people-->
+<!--TODO: Have actual locations and times for Bethel shuttle. Also actual waitlist numbers-->
+{% block body_content %}
+
+    <div class="wrapper">
+        <h4>Select a location and time below to request the shuttle:</h4>
+    </div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+
+    <div class="wrapper">
+        <div class="dropdown">
+            <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Location
+            </button>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                <a class="dropdown-item" href="#">*Insert location here*</a>
+                <a class="dropdown-item" href="#">*Insert more locations*</a>
+            </div>
+        </div>
+
+        <div class="dropdown">
+            <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Time
+            </button>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+               <a class="dropdown-item" href="#">*Insert time here*</a>
+                <a class="dropdown-item" href="#">*Insert more times here*</a>
+            </div>
+        </div>
+
+        <div class="submit">
+            <input class="btn btn-warning btn-sm" type="submit" value="Request Shuttle">
+        </div>
+    </div>
+
+    <div class="wrapper">
+       <h4>Students awaiting shuttle: <span class="badge badge-light">*Insert number here*</span></h4>
+    </div>
+
+{% endblock %}
```

Comment:
For the time being you could probably throw all things involving student interaction into the /schedules templates directory.

Reply:
Sounds good I made those changes